package.json

```diff
@@ -60,5 +60,8 @@
       ],
       "extensions": "jpeg,jpg,png,gif,svg"
     }
+  },
+  "dependencies": {
+    "mjml": "^4.11.0"
```

Comment:
I made the call to put this one in dependencies instead of devDependencies. In practice it doesn't really make a difference in this case, but the rule of thumb I follow is whether the dependency need only be available in the build environment or also in the runtime environment. As presently architected we're invoking mjml in the runtime environment (though if we actually had a build process in between this and runtime there's no reason the mjml couldn't be compiled ahead of time) so I'd suggest it stay where it is for now